serverless.yml

```diff
@@ -30,6 +30,14 @@ functions:
           method: GET
           authorizer: auth
           cors: true
+  get-unassigned-users:
+    handler: 'src/api/projects.unassigned'
+    events:
+      - http:
+          path: /project/{id}/users_not_in
```

Comment:
Path should be changed to `/projects/{id}/users_not_in`

Reply:
Will change it.